resources/netlog_defs.ts

```diff
@@ -1118,7 +1138,6 @@ const latestLogDefinitions = {
       key: 4,
       value: 5,
     },
-    globalInclude: true,
```

Comment:
What do you think about leaving this as `globalInclude: true`. My reasoning for setting this was mostly is so that if folks split/anonymize logs and then include them in the contribute logs file, then we'll always have the rsv lines if needed.

Reply:
> What do you think about leaving this as `globalInclude: true`. My reasoning for setting this was mostly is so that if folks split/anonymize logs and then include them in the contribute logs file, then we'll always have the rsv lines if needed.

So my thinking on removing this was two-fold.  First, they aren't really global includes since they are zone-specific and sent on zone-in.  Second, you can have multiple zone-ins (for that matter, multiple fights) in the same log file, and globalIncludes will export all of them in the split log file.  So if you end up trying to split a log file with a lot of prog hours (particularly across multiple fights), you'll end up with a lot of duplicate rsv line,ms (and possibly some irrelevant ones).

The logic I added to splitter will still export all of the rsv lines for a particular zone in the split/anonymized log, just like we do for combatants (which I think is analogous).  But no dups and no stuff from other zones.